LoopUI/Extensions/DismissibleHostingController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  DismissibleHostingController.swift
```

Comment:
[nit] consider naming this `DismissibleHostingController+<purpose>.swift` where `<purpose>` is the purpose of this extension.  (not blocking)